hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -1295,6 +1302,7 @@ protected RequestFactory createRequestFactory() {
         .withContentEncoding(contentEncoding)
         .withStorageClass(storageClass)
         .withMultipartUploadEnabled(isMultipartUploadEnabled)
+        .withPartUploadTimeout(partUploadTimeout)
```

Comment:
How is this part upload timeout different that multipart upload timeout?

Reply:
its my new option, same value for simple PUT as multipart; we patch the individual requests

Reply:
anyway
1. will cut
2. will modify info log to only print once per stream, to keep that log noise down.